<launch>
  <!-- Levanta el nodo base para la comunicacion MCC <--> Serial -->
  <node name="pc2ioboard" pkg="dm3_core" type="pc2ioboard"/>
 
  <!-- Levanta los modulos -->
  <node name="io_admin_module" pkg="dm3_core" type="io_admin_module.py"/>
  <node name="io_motor_module" pkg="dm3_core" type="io_motor_module.py">
      <param name="rated_state" type="bool" value="False" />
  </node>

  <node name="io_dm3_module"   pkg="dm3_core" type="io_dm3_module.py"/>
</launch>
